pre/remap_restart/remap_utils.py

```diff
@@ -1,14 +1,286 @@
 #!/usr/bin/env python3
 #
+# remap_restarts package:
+#   remap_utils.py contains utility functions and definitions
+#
 import os
 import subprocess
 import ruamel.yaml
 from collections import OrderedDict
 import shutil
 import questionary
+import glob
+import shlex
+import netCDF4 as nc
+
+# shared global variables
+
+# top-level directory for BCs (machine-dependent)
+
+choices_bc_base  =[ "NCCS/Discover : /discover/nobackup/projects/gmao/bcs_shared/fvInput/ExtData/esm/tiles",
```

Comment:
@weiyuan-jiang : Thanks for adding the question about bc_base.  However, don't we want to do this a bit differently?  That is, first detect if we are on Discover or at NAS, and automatically select the hardwired bc_base dir accordingly, without bothering the user.  The question is needed only when the user is not at NCCS or NAS.  I suppose we can fix this later, as it does not impact the yaml interface and only touches on the user experience.  But then again, it might be easier to fix it now while we're at it.

Reply:
I think this provide the users an opportunity  to specify their own bc_base.